project_event/models/project_task.py

```diff
@@ -39,6 +39,7 @@ class Task(models.Model):
     category_id = fields.Many2one(
         'task.category',
         string='Category',
+        default=lambda self: self.get_default_category()
```

Comment:
You can put it directly : default=lambda self: self.env['task.category'].search([('is_default', '=', True)], limit=1)

Reply:
I checked with @wbeverly we can use the boolean field is_default but we must display it in the form view (like that the admin can select the default category) and also we must add a constraint to only allow one true vale for the is_default field

Reply:
Done